### PR TITLE
Arthur replication

### PR DESCRIPTION
**Pull request by ArthurChen189**
OS: Ubuntu 18.04 bionic
Results are the same:
passage:
```
map                   	all	0.1957
recall_1000           	all	0.8573
```

doc:
```
map                   	all	0.2789
recall_1000           	all	0.9326
```

### CODE REVIEW

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -212,3 +212,4 @@ That's it!
 + Results replicated by [@KaiSun314](https://github.com/KaiSun314) on 2021-01-08 (commit [`113f1c7`](https://github.com/castorini/anserini/commit/113f1c78c3ffc8681a06c571901cf9ad8f5ee633))
 + Results replicated by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`179c242`](https://github.com/castorini/anserini/commit/179c242562bbb990e421f315370f34d4d19bbb9f))
 + Results replicated by [@larryli1999](https://github.com/larryli1999) on 2021-01-22 (commit [`3f9af5`](https://github.com/castorini/anserini/commit/3f9af54d6215eacbade7fc99ff8890920fdddee0))
++ Results replicated by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-04-08 (commit [`d43b5c6`](https://github.com/castorini/anserini/commit/b0e0092fb4c4f2bf053c9e5d35b63aca6fd30e47))
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:215]: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

File: docs/experiments-msmarco-passage.md [hunk 2]
```diff
@@ -189,3 +189,4 @@ To replicate these results, the `SearchMsmarco` class above takes `k1` and `b` p
 + Results replicated by [@KaiSun314](https://github.com/KaiSun314) on 2021-01-08 (commit [`113f1c7`](https://github.com/castorini/anserini/commit/113f1c78c3ffc8681a06c571901cf9ad8f5ee633))
 + Results replicated by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`179c242`](https://github.com/castorini/anserini/commit/179c242562bbb990e421f315370f34d4d19bbb9f))
 + Results replicated by [@larryli1999](https://github.com/larryli1999) on 2021-01-22 (commit [`3f9af5`](https://github.com/castorini/anserini/commit/3f9af54d6215eacbade7fc99ff8890920fdddee0))
++ Results replicated by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-04-07 (commit [`6fca6df`](https://github.com/castorini/anserini/commit/6fca6dfb50f446972813f76257a64c06bc1b6903))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:192]: Same here...

Comment (ArthurChen189) [reply docs/experiments-msmarco-passage.md:192]: Will resubmit a pr. Thanks

**Comment by ArthurChen189** [comment]
passage and doc .md updated and ready to be merged.

**Comment by lintool** [comment]
I don't think the "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository." issue has been resolved...

**Comment by MXueguang** [comment]
@ArthurChen189 the commit id in the reproduction log should be: _the last commit ID on the master branch_ (not a commit from a branch in your forked repo.)
i.e.. I guess you are doing replication based on the version at commit (45a5a219af92c82dd429f4d96aee13bd87825147)? 
Could you double check?

**Comment by ArthurChen189** [comment]
> @ArthurChen189 the commit id in the reproduction log should be: _the last commit ID on the master branch_ (not a commit from a branch in your forked repo.)
> i.e.. I guess you are doing replication based on the version at commit ([45a5a21](https://github.com/castorini/anserini/commit/45a5a219af92c82dd429f4d96aee13bd87825147))?
> Could you double check?

Thanks for your suggestion. What I did was that I forked the repo and then created a new branch under my forked repo. I committed to adding "results replicated ..." at the bottom of the ".md" without commit id first,
```
+ Results replicated by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-04-07 
```
and then added the previous commit id to my "results replicated ..." and made another commit.

**Comment by MXueguang** [comment]
Yes, if so, the replication log are expected to be:
```
Results replicated by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-04-08 (commit [`45a5a21`](https://github.com/castorini/anserini/commit/45a5a219af92c82dd429f4d96aee13bd87825147))
```

Our purpose of replication log is to record the latest version of our main repo that confirmed "working"

**Comment by ArthurChen189** [comment]
I see, then should I use exactly the same commit id for all replications then?
Thank you so much.

**Review by lintool — APPROVED** [review]

**Pull request merged**